src/frontend/src/components/displayError.ts

```diff
@@ -10,6 +10,12 @@ export type ErrorOptions = {
   primaryButton: TemplateElement;
 };
 
+export const WEBAUTHN_CANCEL_TEMPLATE = {
+  title: "Operation canceled",
+  message:
+    "The interaction with your security device was canceled or timed out. Please try again.",
+};
+
```

Comment:
This is not the right place for this; displayError is a component that should be error-agnostic. How about `webAuthnErrorUtils`?